src/scicookie/{{cookiecutter.project_slug}}/virtualenvs/conda/dev.yaml

```diff
@@ -5,6 +5,9 @@ channels:
 dependencies:
   - python >=3.8.1,<4
   - pip
+  - pip:
```

Comment:
this `pip:` section should be at the end of the file and should be included just in the case that the build_system is poetry and the doc engine is mkdocs